src/toil/jobStores/abstractJobStore.py

```diff
@@ -94,18 +94,18 @@ def getJobStoreClasses():
     jobStoreClassNames = (
         "toil.jobStores.azureJobStore.AzureJobStore",
         "toil.jobStores.fileJobStore.FileJobStore",
+        "toil.jobStores.googleJobStore.GoogleJobStore",
         "toil.jobStores.aws.jobStore.AWSJobStore",
         "toil.jobStores.abstractJobStore.JobStoreSupport"
     )
-
     jobStoreClasses = []
     for className in jobStoreClassNames:
         moduleName, className = className.rsplit('.', 1)
         from importlib import import_module
         try:
             module = import_module(moduleName)
         except ImportError:
-            logger.info("Unable to import '%s'" % moduleName)
+            logger.info("Unable to import '%s'. This may be because its dependencies are not installed." % moduleName)
```

Comment:
You probably want to mention the `extra` mechanism here. And please wrap your code at column 100.